command_line.py

```diff
@@ -0,0 +1,61 @@
+"""Main functions to simulates reads"""
```

Comment:
this file should be removed, the file is the one in mirtop/command_line.py

Reply:
Hi @lpantano, I removed the redundant files. Please check it once. Thank you.